Automattic/jetpack
# Story block: Add more media options to choose from in the editor

#### Changes proposed in this Pull Request:
This change allows for selecting media in the story block from the same media sources as the gallery block and other visual blocks:

![image](https://user-images.githubusercontent.com/230230/116422687-45f52880-a840-11eb-97dd-13e756dd0cdc.png)



#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:
- Apply this patch on a jetpack site connected to wordpress.com
- Navigate to a post containing a story or create a new story
- Click on Select Media
- Notice Google Photos and Pexels are available and that we can mix media from different sources when editing a story
